CopyKeys/create_aad_app_key.ps1

```diff
@@ -0,0 +1,119 @@
+Param(
+
+  [Parameter(Mandatory = $true,
+
+             HelpMessage="Identifier of the Azure subscription to be used")]
+
+  [ValidateNotNullOrEmpty()]
+
+  [string]$subscriptionId,
+
+
+  
+  [Parameter(Mandatory = $true,
+
+             HelpMessage="Name of the AAD application that will be used to write secrets to KeyVault. A new application with this name will be created if one doesn't exist. If this app already exists, pass aadClientSecret parameter to the script")]

```

Comment:
From help message: "pass aadClientSecret parameter to the script"
are we suggesting customers to pass aadClientSecret  with aadAppName?